corebehrt/functional/features/creators.py

```
@@ -47,63 +48,110 @@ def create_age_in_years(concepts: pd.DataFrame) -> pd.DataFrame:
+    Returns:
+        DataFrame with patient information including birthdate, deathdate, and background variables
+    """
+    # Get unique patients
+    patients = concepts[PID_COL].unique()
+
+    # Initialize patient info - handle empty case
+    patient_info = pd.DataFrame({PID_COL: patients})
+
+    # If no patients, return empty DataFrame with proper structure
+    if len(patients) == 0:
+        warnings.warn("No patients found in concepts")
```

Ruff (0.11.9)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)